File: generators/kotlin/testdata/todo_method_types.kt
@Serializable
data class AddItemInput(
    @SerialName("item") val item: Item = Item(),
)

@Serializable
class AddItemOutput(
)

@Serializable
class GetItemsInput(
)

@Serializable
data class GetItemsOutput(
    @SerialName("items") val items: List<Item> = emptyList(),
)

@Serializable
data class RemoveItemInput(
    @SerialName("id") val id: Long = 0,
)

@Serializable
data class RemoveItemOutput(
    @SerialName("item") val item: Item = Item(),
)

File: generators/golang/testdata/todo_method_types.go
type AddItemInput struct {
	// the item to add.
	Item todo.Item `json:"item"`
}

type AddItemOutput struct {
}

type GetItemsInput struct {
}

type GetItemsOutput struct {
	// Items is the list of to-do items.
	Items []todo.Item `json:"items"`
}

type RemoveItemInput struct {
	// the id of the item to remove.
	Id int `json:"id"`
}

type RemoveItemOutput struct {
	// the item removed.
	Item todo.Item `json:"item"`
}
